R/opq.R

```diff
@@ -310,9 +310,9 @@ paste_features <- function (key, value, key_pre = "", bind = "=",
 #'     add_osm_feature (key = "highway", value = "!primary")
 #'
 #' # key negation without warnings
-#' q3 <- opq ("Vinçà", osm_type="node") |>
-#'     add_osm_feature (key = c("name", "!name:ca"))
-#' q4 <- opq ("el Carxe", osm_type="node") |>
+#' q3 <- opq ("Vinçà", osm_type = "node") |>
```

Comment:
Nice, thanks for finding these!! :+1:

Reply:
The merit goes for pre-commit & styler 😋